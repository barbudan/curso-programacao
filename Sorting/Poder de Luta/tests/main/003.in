5
a 10
c 30
b 30
d 90
e 5