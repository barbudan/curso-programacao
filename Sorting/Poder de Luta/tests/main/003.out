d 90
b 30
c 30
a 10
e 5